pkg/config/runtime_config.go

```diff
@@ -82,13 +84,22 @@ func BuildRestConfig(rtCfg RuntimeConfig) (*rest.Config, error) {
 
 // BuildRuntimeOptions builds the options for the controller runtime based on config
 func BuildRuntimeOptions(rtCfg RuntimeConfig, scheme *runtime.Scheme) ctrl.Options {
+	// if DefaultNamespaces in Options is not set, cache will watch for all namespaces
```

Comment:
I think I am misreading this. We want VPC RC client to watch all namespaces, right? Since this is used by pod webhook

Reply:
We don't set namespace here unless users deploy the controller to dataplane and specifically set the flag `rtCfg.WatchNamespace` otherwise it is default to `corev1.NamespaceAll` (https://github.com/aws/amazon-network-policy-controller-k8s/blob/6d4852c37dfa8c52ad90c8d59e4207f3d6e553f3/pkg/config/runtime_config.go#L61). So in general case, it is all namespaces.

Reply:
Ah, I was confusing which repo this was